blockchain/blockchain.go

```diff
@@ -277,26 +253,6 @@ func (b *Blockchain) Receipt(hash *felt.Felt) (*core.TransactionReceipt, *felt.F
 	return receipt, blockHash, blockNumber, b.database.View(func(txn db.Transaction) error {
 		var err error
 		receipt, blockHash, blockNumber, err = receiptByHash(txn, hash)
-
-		// not found in the canonical blocks, try pending
-		if errors.Is(err, db.ErrKeyNotFound) {
-			var pending Pending
-			pending, err = b.pendingBlock(txn)
-			if err != nil {
-				return err
-			}
-
-			for i, t := range pending.Block.Transactions {
-				if hash.Equal(t.Hash()) {
-					receipt = pending.Block.Receipts[i]
-					blockHash = nil
-					blockNumber = 0
-					return nil
-				}
-			}
-			return db.ErrKeyNotFound
-		}
-
```

Comment:
why for tx-s you moved logic to `transactionByHash` but here it's not part of receiptByHash anymore ?

Reply:
I don't think I understand your question.

I removed checking of pending transactions and receipt from both `TransactionByHash` and `Receip`

Reply:
Sorry, seems like I was referring to some other commit. I don't see much differences now.